catalog/task/add-fbc-fragment-to-index-image/0.1/README.md

```diff
@@ -0,0 +1,16 @@
+# add-fbc-fragment-to-index-image
+
+Calls IIB service to submit a build request to add fbc fragments to an index image
+
+## Parameters
+
+| Name | Description | Optional | Default value |
```

Comment:
Is there a reason why all the parameters are capitalized here but not for the pipelines?